lib/nsxv.py

```diff
@@ -326,14 +329,25 @@ def __str__(self):
     for s in services:
       service = '%s%s' % (service, s)
 
+    # applied_to
+    appliedto_list = ''
+    if self.applied_to:
+      appliedto_list = '<appliedToList>'
+      appliedTo = '%s%s%s' % (_XML_TABLE.get('appliedToStart'),
```

Comment:
Looks like this one was missed. Variable names should be lower case and underscore separated.